astroplan/tests/test_core.py

```diff
@@ -155,3 +155,34 @@ def test_FixedTarget_ra_dec():
                                                         'SkyCoord')
     assert vega.coord.dec == vega_coords.dec == vega.dec, ('Retrieve Dec from '
                                                            'SkyCoord')
+
+def test_parallactic_angle():
+    lat = '+19:00:00'
+    lon = '-155:00:00'
+    elevation = 0.0 * u.m
+    location = EarthLocation.from_geodetic(lon, lat, elevation)
+    time = Time('2015-01-01')
+    vega = SkyCoord('18h36m56.33635s', '+38d47m01.2802s')
+    obs = Observer(name='Observatory', location=location)
+    astroplan_q = obs.parallactic_angle(time, vega)
+    threshold_angle = 2*u.deg
```

Comment:
Honestly I don't know either, but I'm not sure I would trust PyEphem as an authoritative source, though - less clear to me that that's well-tested/used (it's based on XEphem, and my subjective impression is that that is less likely to be used by people who care about the pang).  Maybe instead one of the observatory-based calculators?  If you google "parallactic angle calculator" you get things from ESO, Keck, and the like (although with various levels of inputs and usability).  Or if there's an IDL code in one of the well-used libraries, that might be good too.